<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Pack</class>
 <widget class="QWidget" name="Form_Pack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,4,1,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>QT版本</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_QT_installVer"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>编译器版本</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_QT_compileVer"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_path">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,4">
       <item>
        <widget class="QPushButton" name="pushButton_create">
         <property name="text">
          <string>生成</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_moveRelease">
         <property name="text">
          <string>移动整理</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_about">
         <property name="text">
          <string>关于</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
